about-me.html

```diff
@@ -0,0 +1,56 @@
+<html>
+<head>
+	<title>Saskia West Software Dev Portfolio</title>
+	<link href="normalize.css" rel="stylesheet" type="text/css">
+	<link href="style.css" rel="stylesheet" type="text/css">
+	<link href="https://fonts.googleapis.com/css2?family=Cinzel&family=Girassol&family=Gloria+Hallelujah&family=Montserrat:wght@200&family=Open+Sans+Condensed:wght@300&display=swap" rel="stylesheet">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+</head>
+
```

Comment:
keep an eye on the empty lines,
you should only have them where you think it's necessary - like marking where the sections start and end etc.